### PR TITLE
update faq page

### PR DESCRIPTION
**Pull request by nataliejeremy**
This PR closes issue #652 CSE please review--thanks!

### CODE REVIEW

File: source/docs/articles/frequently-asked-questions.md [hunk 1]
```diff
@@ -9,25 +9,27 @@ keywords: getting started, faqs, sites, pantheon, plans, developing, security
...
 
-Pantheon is free for developers. Our live site plans currently start as low as $25 monthly for personal sites, and $100 for professional sites. Learn more on [our pricing page](https://getpantheon.com/pricing).
+Pantheon is free for developers. Our live site plans currently start as low as $25 monthly for personal sites, and $100 for professional sites. Learn more on our [pricing page](https://getpantheon.com/pricing).
 
 ### Where are the Pantheon servers located?
 
 All Pantheon servers are currently located in the United States. We have plans to expand to Europe, but we don't have an ETA for when they will be available for end-users.
 
-### Can I run non Drupal applications on Pantheon?
+You can use a [CDN](/docs/articles/drupal/content-delivery-network-cdn-for-file-distribution/) for rapidly serving files from multiple locations, and Pantheon is faster than the time savings from having a local server.
```
Comment (ari-gold) [inline source/docs/articles/frequently-asked-questions.md]: What does this mean "...and Pantheon is faster than the time savings from having a local server."

Comment (nataliejeremy) [inline source/docs/articles/frequently-asked-questions.md]: @ari-gold Zack wanted that added here.

File: source/docs/articles/frequently-asked-questions.md [hunk 2]
```diff
@@ -35,7 +37,7 @@ This is not officially supported, but the PHP runtime is complete. Some users ha
 
 Pantheon supports toggling between local development mode using `git push` to transfer all code changes, and an on-server development mode, which provides access to the codebase via SFTP.
 
-Direct SSH access is not supported, but you are able to directly interface with mysql, use CLI tools ([Terminus](https://github.com/pantheon-systems/cli), [drush](/docs/articles/local/drush-command-line-utility/), [WP-CLI](#does-pantheon-support-wp-cli?)), and SFTP files. Rsync is currently unstable on Pantheon. The engineering team is at work to improve its functionality on the platform.
+Direct SSH access is not supported, but you are able to directly interface with mysql, use CLI tools ([Terminus](https://github.com/pantheon-systems/cli), [drush](/docs/articles/local/drush-command-line-utility/), [WP-CLI](#Does-Pantheon-support-WP-CLI?), and SFTP files. Rsync is currently unstable on Pantheon. The engineering team is at work to improve its functionality on the platform.
```
Comment (ari-gold) [inline source/docs/articles/frequently-asked-questions.md]: I thought Rsync works well as long as you are doing it to a non-live env and make sure to use the flags that are in our Rsync article?  /cc @suzannealdrich @newtoid @timani

Comment (suzannealdrich) [inline source/docs/articles/frequently-asked-questions.md]: Rsync is a best practice, it is not unstable, it just creates temporary dotfiles faster than we can store and delete them, sometimes resulting in "ghost" files that aren't in Valhalla but are in fusedav. Workaround is to not store temp files in Valhalla during transmission, thus the required flag `--temp-dir=../tmp/`.

**Comment by nataliejeremy** [comment]
@bmackinney Please review

**Comment by nataliejeremy** [comment]
@bmackinney Please review and merge.

**Comment by bmackinney** [comment]
:+1:

**Pull request merged**